providers.tf

```diff
@@ -5,4 +5,5 @@ terraform {
       version = ">= 4.0"
     }
   }
+  required_version = ">= 1.4.0"
```

Comment:
This seems like a leftover of the `terraform_data` removal. I would recommend removing it too.

Reply:
```suggestion
```

Reply:
Thanks, done